### DISCUSSION TITLE
Chart Axis - Display unit (Thousands, Millions)

### DISCUSSION THREAD

**Discussion by Matteo-pd** [PHPOffice/PhpSpreadsheet#3656] [General]
I have an Excel Chart with large values like 300,000.0
In Excel, I have the Axis Display Unit set to "Millions", so my Axis Label has values like 0.1; 0.2; 0.3; ... 

How can I read the Axis Display Unit (None, Thousands, Millions, etc.)?

![Screenshot 2023-07-28 at 09 35 49](https://github.com/PHPOffice/PhpSpreadsheet/assets/135317660/48eb9a7b-f023-49bd-aa98-d0e3aa15d422)

**Comment by oleibman** [comment]
You can't yet. I will look into what needs to be done to support it.

**Comment by oleibman** [comment]
Converted to issue #3833. Closing discussion. Please continue the conversation as needed in the issue.
